scripts/build_cloud_build_image.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+pushd cloudbuild/builder
+
+# Get latest VS Code package
+wget 'https://code.visualstudio.com/sha/download?build=stable&os=linux-deb-x64' -O vscode.deb
```

Comment:
I'm not very familiar with cloud build, but could we have more images and run in multiple different environments (covering multiple OS's and architectures) this may be a test we run asynchronously.

Reply:
We can have a variety of images, but running Windows or macOS for testing would not be easy given the Docker on Linux VM underpinnings of Cloud Build.